### PR TITLE
add JupyterLab module

### LINKED ISSUE #9: Jupyter
## Must have 

- [ ] Auto-installs Jupyter with pip 
- [ ] Starts JupyterLab
- [ ] `coder_app`
- [ ] `coder_script`

## Should have

- [ ] Variable to specify version
- [ ] Variable to specify whether they want hub or lab

### PR DESCRIPTION
**Pull request by matifali**
closes #9

### CODE REVIEW

File: .sample/run.sh
Comment (matifali) [inline .sample/run.sh:1]: chmod +x

File: new.sh
Comment (matifali) [inline new.sh:1]: we should explicitly `chmod +x` the `run.sh`

File: .images/jupyterlab.webp
Comment (matifali) [inline .images/jupyterlab.webp:1]: Let us put all screenshots in .images so that when someone uses the module, they don't pull the images.

Comment (bpmct) [reply .images/jupyterlab.webp:1]: totally agree! was going to mention this and forgot

**Review by bpmct — APPROVED** [review]

**Pull request merged**